Makefile

```diff
@@ -40,8 +46,9 @@ build-yarn-watch: build-solidity-types build-cargo
 
 .PHONY: build-cargo
 build-cargo: ## build cargo packages and create boilerplate JS code
-	cargo build --release --target wasm32-unknown-unknown
+ifeq ($(origin NO_CARGO),undefined)
```

Comment:
Why is this needed?

Reply:
added a comment

Reply:
I know what it does, but at which point would we want to skip that step?